assets/src/setup/pages/template-mode/get-selection-details.js

```diff
@@ -47,7 +48,7 @@ export function getRecommendationLevels( { userIsTechnical, hasPluginIssues, has
 		return {
 			[ READER ]: MOST_RECOMMENDED,
 			[ STANDARD ]: RECOMMENDED,
-			[ TRANSITIONAL ]: RECOMMENDED,
+			[ TRANSITIONAL ]: currentThemeIsReaderTheme ? MOST_RECOMMENDED : RECOMMENDED,
```

Comment:
@westonruter should we update the description for transitional mode if the theme is AMP compatiable? Currently, it recommends both transitional and reader mode, but does not inlcude the fact that the user's current theme is AMP compatible:

![image](https://user-images.githubusercontent.com/16200219/86493741-3745fe00-bd62-11ea-9569-8998270f9d02.png)

Reply:
Yes, that's a good point. We should export to JS this logic from PHP:

```php
$has_amp_support = current_theme_supports( 'amp' ) || in_array( get_stylesheet(), AMP_Core_Theme_Sanitizer::get_supported_themes(), true );
```

Note I'm intentionally using `get_stylesheet()` here as opposed to `get_template()` because child themes of core themes are often not AMP-compatible.

Nevertheless, we may want to hold off on doing this until I finish my work on the Reader theme loading because this changes how the `amp` theme support is mutated (or not).

There's also a case to do something similar for Reader mode, for themes that _don't_ have `amp` theme support and yet have an `amp` directory, which is captured by this logic at the moment:

https://github.com/ampproject/amp-wp/blob/a8a4d4d05f9727aad2304db103a3ab97817aafa6/includes/class-amp-theme-support.php#L349-L366

We may want to mention that the active theme has special legacy Reader mode support, and also indicate that in the list of Reader themes.

But I suggest filing this in a new issue.

Reply:
https://github.com/ampproject/amp-wp/issues/4990